pkg/scheduler/framework/plugins/nodeaffinity/node_affinity.go

```diff
@@ -147,7 +147,7 @@ func (pl *NodeAffinity) Score(ctx context.Context, state *framework.CycleState,
 
 	node := nodeInfo.Node()
 	if node == nil {
-		return 0, framework.AsStatus(fmt.Errorf("getting node %q from Snapshot: %w", nodeName, err))
+		return 0, framework.AsStatus(fmt.Errorf("getting node %q from Snapshot: node not found", nodeName))
```

Comment:
I feel we cannnot even get here, the whole block here could be removed. 

```
	if v, ok := s.nodeInfoMap[nodeName]; ok && v.Node() != nil {
		return v, nil
	}
```

Reply:
Actually, there are similar codes in many places. If to remove it, it’s best to remove it all at once.

Reply:
Indeed, there is no need to check “node == nil” here.

Reply:
Already fixed. whole block has been removed. 
 I will submit another pr to fix other places later.